eth-bytecode-db/eth-bytecode-db-proto/proto/v2/eth-bytecode-db.proto

```diff
@@ -223,6 +231,9 @@ message SearchAllSourcesRequest {
   string chain = 3;
   /// The address of the contract being verified
   string address = 4;
+  /// If set and is `false`, will not try to obtain sources from 3rd party services
+  /// (e.g., Sourcify and Etherscan)
+  optional bool only_local = 5;
```

Comment:
if `only_local=false` it **will not** go to Sourcify and Etherscan ? maybe it **will** go?

Reply:
and why `optional`? you need default value to be `true`? in this case let's just reverse meaning of variable to be `search_remote`

Reply:
Nope, the default value is set to be `false`, you can find it in the endpoint handler further. I've added `optional` here merely so to show that users may not specify that value by default. In which case, it will default to `false` and the behaviour will not change from the current one

Reply:
Do you suggest to remove optional, and set it to default to `false` in case if missing in the `build.rs`?

Reply:
oh, I see. Well, actually I don't know what is better, maybe writing `optional` to show that this field is actually optional is better than add `serde::default` in `build.rs` :)